apps/etterlatte-saksbehandling-ui/client/src/components/behandling/aktivitetsplikt/Aktivitetsplikt.tsx

```diff
@@ -87,20 +88,41 @@ export const Aktivitetsplikt = (props: { behandling: IDetaljertBehandling }) =>
           <Heading level="1" spacing size="medium">
             Gjenlevende sin situasjon
           </Heading>
-          <BodyLong spacing>
-            Det stilles ulike krav til aktivitet utifra tid etter dødsfallet. Seks måneder etter dødsfallet må
-            gjenlevende være i minst 50 % aktivitet for å ha rett til omstillingsstønad. Videre kan det stilles krav til
-            100 % aktivitet etter 12 måneder. I visse tilfeller kan man ha rett på omstillingsstønad selv om
-            aktivitetskravet ikke er oppfylt.
-          </BodyLong>
+          <ReadMore header="Slik vurderer du gjenlevendes situasjon">
+            <BodyLong spacing>
+              Det stilles ulike krav til aktivitet basert på tiden som har gått etter dødsfallet. Seks måneder etter
+              dødsfallet må den gjenlevende være i minst 50 % aktivitet for å ha rett til omstillingsstønad. Videre kan
+              det kreves 100 % aktivitet etter tolv måneder. I enkelte tilfeller kan man likevel ha rett til
+              omstillingsstønad, selv om aktivitetskravet ikke er oppfylt.
+            </BodyLong>
+            <BodyLong spacing>
+              Selv om de første seks månedene er fritatt, er det viktig å kartlegge situasjonen tidlig for å sikre
+              riktig oppfølging og forberede brukeren på kravene som følger
+            </BodyLong>
+          </ReadMore>
         </TekstWrapper>
 
-        {isValidDateOfDeath(avdoedesDoedsdato!!) && (
-          <AktivitetspliktTidslinje behandling={behandling} doedsdato={avdoedesDoedsdato!!} />
+        {isValidDateOfDeath(avdoedesDoedsdato!!) && ( // todo denne valideringen burde skje et annet sted vel og si noe om den ikke er gyldig
```

Comment:
Jeg tenker det er kanskje vært en jira på dette (gjelder vel også linje 113 lengre nede) om at vi må ha noe overordnet validering på dødsdato er satt. 

Sikkert se på det i sammenheng med tidligere familiepleier, der vi må ha dødsdato for OMS ikke tidl. fam. pleier, og for tidligere familiepleier er det dato pleieforholdet opphørte som burde bli sett på / sendes med.

Det er nok lurt å ha noe felles "uthenting" / kilde til sannhet om hvilken dato vi skal bruke andre steder i businesslogikk, og det er kanskje også noe å se på i backend? men det går litt utenfor akkurat denne endringen

Reply:
Enig, https://jira.adeo.no/browse/EY-4939